src/icrar/leap-accelerate-cli/main.cc

```diff
@@ -150,32 +193,41 @@ int main(int argc, char** argv)
     {
         return app.exit(e);
     }
-    ArgumentsValidated args = ArgumentsValidated(std::move(rawArgs));
-
-    std::cout << "running LEAP-Accelerate:" << std::endl;
 
-    auto queue = std::queue<casalib::Integration>();
-
-    switch(args.GetComputeImplementation())
-    {
-    case ComputeImplementation::casa:
-    {
-        std::vector<casacore::MVDirection> directions;
-        icrar::casalib::Calibrate(args.GetMeasurementSet(), directions, 16001);
-        break;
-    }
-    case ComputeImplementation::eigen:
+    try
     {
-        std::vector<icrar::MVDirection> directions; //ZenithDirection(ms);
-        icrar::cpu::Calibrate(args.GetMeasurementSet(), directions, 16001);
-        break;
+        ArgumentsValidated args = ArgumentsValidated(std::move(rawArgs));
```

Comment:
The `std::move` doesn't seem necessary given that the `ArgumentsValidated` constructor takes a const ref, not an rvalue. `std::move` could be removed here, or the `ArgumentsValidated` constructor could be changed to accept an rvalue instead.

Reply:
I think I'll update to an rvalue here as my intention for using std::move is so the reader will know not to use rawArgs once the validated arguments object exists